providers/sms77/src/lib/sms77.provider.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  ChannelTypeEnum,
+  ISendMessageSuccessResponse,
+  ISmsOptions,
+  ISmsProvider,
+} from '@notifire/core';
+
+import Sms77Client, {SmsJsonResponse, SmsParams} from 'sms77-client'
```

Comment:
```suggestion
import Sms77Client, { SmsJsonResponse, SmsParams } from 'sms77-client';
```

Could you please run prettier on this file? 💅

Reply:
Just did that, should be fine now.
Maybe it could be helpful to add prettier to the pre-commit hook?

Reply:
@matthiez there should be a lint-staged pre-commit hook that does prettier. Looks like it's not working :( Could you open an issue for that?

Reply:
I'm playing around a bit for now trying to fix this. I have never really used lint-staged so there is that. It seems to pick up the configuration from notifire/libs/sdk/package.json and not the actual package.json from the project.

Besides this issue, I noticed that the provider generator generated .eslintrc.js - shouldn't it be .eslintrc.json like with the existing providers?

Reply:
Yes, it's a mistake. @galezra already fixed that at v0.3.0 hopefully I'll merge it tomorrow or today.